README.md

```diff
@@ -64,14 +73,15 @@ A curated list of awesome TON ecosystem.
 ## Videos
 
 - [Telegram Open Network Tutorial](https://www.youtube.com/watch?v=J7K2nq5lf7I&feature=youtu.be) - How to use light client by [@rstormsf](https://twitter.com/rstormsf)
+- [About TON](https://youtu.be/3O-jnS72gY4)
```

Comment:
Explain, who is owner and name: 

Telegram Open Network – TON (Promo Final Version) by DeCenter